src/hashes/sha2-browser.ts

```diff
@@ -9,11 +9,13 @@ function sha (name: AlgorithmIdentifier): (data: Uint8Array) => Promise<Uint8Arr
 export const sha256 = from({
   name: 'sha2-256',
   code: 0x12,
-  encode: sha('SHA-256')
+  encode: sha('SHA-256'),
+  maxDigestLength: 32
 })
 
 export const sha512 = from({
   name: 'sha2-512',
   code: 0x13,
-  encode: sha('SHA-512')
+  encode: sha('SHA-512'),
+  maxDigestLength: 64
 })
```

Comment:
should we increase the minDigestLength to 32 here, to get 128-bits of collision security?